### PR TITLE
Oh the choices

### PR DESCRIPTION
**Pull request by caiblack**
Please review this, @casey-collab

Things take a serious turn. #raccoons #dinosaurs

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab can you take a look?

File: english/lunge-for-the-lighter/steep-slope/steep-slope.md [hunk 1]
```diff
@@ -5,3 +5,17 @@ this opening before...while being chased by marshmallow men who were shooting
...
+You decide against just diving blindly into the opening, and, in a flash of what could be called intuition, you 
+punch the button. This results in two other openings being revealed to the right of the steeply sloping one. You 
+pause. The first of these new openings reveals a tunnel that ascends slightly, and curves gently away to the right.
+Oddly, it seems to made of stone, unlike the rest of what you have seen of this strange place. You can hear the 
+faint sound of dripping liquid coming from the tunnel. It is lit from above, every two meters or so, by small
+lights with wires connecting each of them The second new opening reveals what appears to be a hallway, lit in the 
+same manner as the first, lined with pictures of various types of bird-like lizards wearing very ... um, unique
+clothing. You can just barely hear odd animalistic noises from some point far down the hallway.
+
+[Walk into the stone tunnel](stone-tunnel)
```
Comment (casey-collab) [inline english/lunge-for-the-lighter/steep-slope/steep-slope.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/lunge-for-the-lighter/steep-slope/steep-slope.md [hunk 2]
```diff
@@ -5,3 +5,17 @@ this opening before...while being chased by marshmallow men who were shooting
...
+
+[Walk into the hallway](hallway)
```
Comment (casey-collab) [inline english/lunge-for-the-lighter/steep-slope/steep-slope.md]: This link is also invalid.

File: english/lunge-for-the-lighter/steep-slope/hallway/hallway.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+With your first step into the hallway, you realize that the situation is about to become much more serious than you
```
Comment (casey-collab) [inline english/lunge-for-the-lighter/steep-slope/hallway/hallway.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/lunge-for-the-lighter/steep-slope/stone-tunnel/stone-tunnel.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+As you enter the tunnel, you realize that what you thought was the sound of dripping liquid was the sound of two
```
Comment (casey-collab) [inline english/lunge-for-the-lighter/steep-slope/stone-tunnel/stone-tunnel.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by caiblack** [comment]
@casey-collab can you take a look?

**Comment by casey-collab** [comment]
Looks great! :dart:

**Pull request merged**